extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {analyticsForDoc} from '../../../src/analytics';
+import {refreshConfigs} from '../../../ads/_a4a-config';
+import {timerFor} from '../../../src/services';
+
+/**
+ * @typedef {{
+ *   visiblePercentageMin: number,
```

Comment:
Can you add documentation explaining what these mean?  Also would be good if time values indicated milliseconds